### ISSUE TITLE
update expo-template-bare-minimum react-native version 0.59.*

### ISSUE THREAD

**Issue by organic-scholar** [expo/expo-cli#493]
please update bare-minimum template to use react-native 0.59.*
0.59 is major stable release now.

**Comment by brentvatne** [comment]
we will update it when we update expo to 0.59 :)

**Comment by joelgetaction** [comment]
Hi @brentvatne - thanks for all your hard work on expo!  Any idea when expo bare template apps will be upgraded to use 0.59.8?  Expo still uses 0.58.6 which is now almost 3 months old.  I'm considering upgrading to 0.59.8 from a an expo bare app, but the RN team made a lot of changes to the cli and deprecations - any experience of that upgrade, should it just work, or are there complications?

The risk of using an old version of RN like 0.58.6 from expo is that the RN team won't support it that much - their first question is usually "are you on the latest version of RN, if not please upgrade then we can address your bug report" ... So not sure what to do - don't want to destabilize my expo app by upgrading if it's not officially supported by the expo team, but also don't want to lose support from the RN team because the expo version of RN is far behind.

I'm happy to help however I can with some guidance.  :-)

Thanks!

**Comment by Luckygirlllll** [comment]
I'm also confused about upgrading to 0.59, I'm using 0.57 in expoKit project, when expo will support 0.59? Will it brake a project if I will upgrade it to 0.59?

**Comment by Kyniska** [comment]
Hi, I'm writing having run into problems with version mismatches that now render my expo Ops unrenderable, and want to express my deep disappointment with this situation.  A few days ago I happened across a React Conference from 2018, in which 'useState' was introduced to solve multiple emerging problems.  The speaker also noted that relying on classes for defining state interferes with the optimization processes of certain devices.  This was from 2018.  "This is pretty important." I thought, "How do I get access, cause I need this?"  Some "fun" adventures revealed I needed to upgrade from 0.57.1 to .59.  OK.  Then I was at 0.60 and problems have refused to leave.  I don't know what the expo team's definition of responsibility is, but, at present, I'm praying 0.59 is supported as I search for the maximum tolerated spec.  This seems no small issue for the community, on something that emerged in 2018.
